zebra/if_netlink.c

```diff
@@ -66,6 +66,12 @@
 
 extern struct zebra_privs_t zserv_privs;
 
+/* Helper function to free altname strings */
+static void altname_free(void *data)
```

Comment:
Why is this method in this file (netlink)?

Reply:
oops, missed this, isn't used anymore.